File: backend/Dtos/AppointmentDto.cs
using LogopedicBackend.Enums;

namespace LogopedicBackend.Dtos;

public class AppointmentDto
{
    public required int Id { get; set; }
    public required DateTimeOffset StartTime { get; set; }
    public required int DurationInMinutes { get; set; }
    public required AppointmentType Type { get; set; }
    public required AppointmentStatus Status { get; set; }
}

File: frontend/src/app/core/models/AppointmentDto.ts
import { AppointmentType } from './AppointmentType';
import { AppointmentStatus } from './AppointmentStatus';

export interface AppointmentDto {
    id: number;
    patientId: number;
    startTime: Date;
    durationInMinutes: number;
    type: AppointmentType;
    status: AppointmentStatus;
}
